tests/ctst/common/common.ts

```diff
@@ -300,40 +300,58 @@ Then('i {string} be able to add user metadata to object {string}',
         }
     });
 
-Then('kafka consumed messages should not take too much place on disk',
+Then('kafka consumed messages should not take too much place on disk', { timeout: 130000 },
```

Comment:
timeshould should be based on `KafkaCleanerInterval`...

Reply:
Should be great but unfortuneatly we can't access `Zenko` object here :(
And I am not sure if this is great anyway, because if we set kafkacleaner interval to 10mn for some reason, we'll end up with 30mn timeout ?